src/VirtualClient/VirtualClient.Main/profiles/MONITORS-GPU.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Description": "Default Monitors",
+  "Parameters": {
+    "MonitorFrequency": "00:01:00",
+    "MonitorWarmupPeriod": "00:01:00"
+  },
+  "Actions": [
```

Comment:
You can remove the Actions and Dependencies sections for the profile. It is cleaner and more consistent with other monitor profiles that way.